libs/content/workspace/src/tab/mod.rs

```diff
@@ -87,13 +87,56 @@ pub enum ClipContent {
     Image(Vec<u8>), // image format guessed by egui
 }
 
-pub trait EventManager {
+// todo: find a better place for the code that attaches additional things to egui::Context
+pub trait ExtendedOutput {
+    fn set_window_title(&self, title: String);
```

Comment:
I think this one exists in egui context: https://docs.rs/egui/latest/egui/struct.Context.html#method.send_viewport_cmd

Reply:
@tvanderstad this is also the API I was alluding to when I said "egui has some basic support for multiple windows"

Reply:
> I think this one exists in egui context: https://docs.rs/egui/latest/egui/struct.Context.html#method.send_viewport_cmd

great feedback